configs/orchestrator/cluster-install-configs/equinix-metal/equinix-metal.vars.envsubst

```diff
@@ -7,7 +7,7 @@ public_eip = "$PUBLIC_EIP"
 
 # Fixed across deployments
 ssh_pubkey            = "$SSH_PUB_KEY"
-metal_project_id      = "$EQUINIX_METAL_PROJECT_ID"
+metal_project_id      = "$PACKET_PROJECT_ID"
```

Comment:
Instead of reverting this maybe we can change the secret from where the env vars are being populated: https://github.com/kinvolk/service-mesh-benchmark/blob/767f72d43ec52868f461e6bd98b7df2612097139/configs/orchestrator/templates/cloud-secrets.yaml#L11 

That way we have equinix metal in both places.